### PR TITLE
changed signature to match getphyllo readme signatures

### PR DESCRIPTION
**Pull request by sumukhaPhyllo**
- Updated readme
- Removed PhylloEnv.development
- Updated PhylloConnect.on(eventName, callback) to phylloConnect.on(eventName, callback)
- Added validators

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "ios": "react-native run-ios",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "start": "react-native start",
-    "test": "jest"
+    "test": "jest",
+    "dev": "cd .. && npm pack && cd example && npm install ../react-native-phyllo-connect-0.0.9.tgz"
```
Comment (akshaybhalotia) [inline example/package.json:12]: Remove the version lock

File: src/PhylloConnect.ts [hunk 2]
```diff
@@ -64,7 +107,16 @@ const PhylloConnectSDK = {
     )
 
     // this is to solely match web sdk signature
-    return { open: () => phyllo.open() }
+    return {
+      open: () => {
+        validateCallbacks(this.callbacksObj)
```
Comment (akshaybhalotia) [inline src/PhylloConnect.ts:112]: Will this still go to `open` if `validateCallbacks` fails?

Comment (sumukhaPhyllo) [reply src/PhylloConnect.ts:112]: @akshaybhalotia

If any error occur, the sdk flow will not open, it'll throw the error instead...

**Comment by sumukhaPhyllo** [comment]
Closed this PR
reason: To follow the new development style
new pr link => https://github.com/getphyllo/phyllo-connect-reactnative/pull/5

**Pull request closed without merging**